xml/Microsoft.VisualBasic.Vsa/VsaEngine.xml

```diff
@@ -456,7 +457,7 @@
         <ReturnType>System.Boolean</ReturnType>
       </ReturnValue>
       <Docs>
-        <summary>To be added.</summary>
+        <summary>Gets a Boolean value that reports whether the engine is currently in run mode.</summary>
         <value>To be added.</value>
```

Comment:
```suggestion
        <value>
          <see langword="true" /> if the script engine is running; otherwise, <see langword="false" />. The default value for this property is <see langword="false" />.</value>
```